EDDiscovery/3DMap/FormMap.cs

```diff
@@ -571,19 +571,34 @@ private void Render()
             GL.Clear(ClearBufferMask.ColorBufferBit | ClearBufferMask.DepthBufferBit);
 
             GL.MatrixMode(MatrixMode.Modelview);
-            GL.LoadIdentity();
-            GL.Rotate(90.0, 1, 0, 0);
+            TransformWorldOrientatation();
+
+            TransformCamera();
+            FlipYAxisOnWorld();
```

Comment:
^This is the only thing that really changed. I just cleaned up a bit by extracting chunks of transformation code into functions as a form of in-built documentation.